java/connector-framework-internal/src/main/java/org/identityconnectors/framework/impl/api/remote/RemoteOperationInvocationHandler.java

```diff
@@ -28,19 +28,15 @@
 import java.lang.reflect.Method;
 import java.util.ArrayList;
 import java.util.List;
+
 import org.identityconnectors.common.CollectionUtil;
 import org.identityconnectors.common.l10n.CurrentLocale;
 import org.identityconnectors.framework.api.RemoteFrameworkConnectionInfo;
 import org.identityconnectors.framework.api.operations.APIOperation;
 import org.identityconnectors.framework.common.exceptions.ConnectorException;
 import org.identityconnectors.framework.impl.api.ObjectStreamHandler;
 import org.identityconnectors.framework.impl.api.StreamHandlerUtil;
-import org.identityconnectors.framework.impl.api.remote.messages.OperationRequest;
-import org.identityconnectors.framework.impl.api.remote.messages.OperationRequestMoreData;
-import org.identityconnectors.framework.impl.api.remote.messages.OperationRequestStopData;
-import org.identityconnectors.framework.impl.api.remote.messages.OperationResponseEnd;
-import org.identityconnectors.framework.impl.api.remote.messages.OperationResponsePart;
-import org.identityconnectors.framework.impl.api.remote.messages.OperationResponsePause;
+import org.identityconnectors.framework.impl.api.remote.messages.*;

```

Comment:
Please restore original imports, we try to avoid star imports.

Reply:
Hi, yes sorry It has to do somehow with mi IDE configuration and I missed it, I will fix this in the next commit.